src/utilities/httpUtils.js

```diff
@@ -0,0 +1,104 @@
+import axios from 'axios';
+
+import { ACCESS_TOKEN } from '~/constant/storage';
+import textConstants from '~/constant/textConstants';
+import * as localStorage from '~/storage/LocalStorage';
+
+/**
+ * Http Get.
+ *
+ * @param {*} url
+ * @param {*} [params={}]
+ * @returns {Promise}
+ */
+export function get(url, params = {}) {
```

Comment:
We can just create an axios object like:
```
const http = axios.create({
  baseURL: 'your://url.com',
  headers: {
    'Content-Type': 'application/json',
  },
});
```

and use it like http.get(url, params) or http.post(url, data)

Reply:
done